conda_forge_tick/deploy.py

```diff
@@ -161,3 +162,7 @@ def deploy(dry_run=False):
             raise RuntimeError("bot did not push its data! stopping!")
     else:
         print("no files to commit!", flush=True)
+
+
+if __name__ == "__main__":
```

Comment:
Do we want to create a second enty point here? I think having both a central enty point with proper help menues and entry points for each "sub-command" is a bit confusing from a user perspective.

Reply:
The main idea behind this was to provide an entry point for local debugging, which is now replaced by the CLI main entry point in bad3bc8f